.travis.yml

```diff
@@ -8,18 +8,15 @@ cache:
   directories:
     - ~/.m2
 
-addons:
-  apt:
-    packages:
-      - groovy
-      - xsltproc
-      - xmlstarlet
-
 branches:
   only:
     - master
 
+before_install:
+  - sudo apt update
+
 install:
+  - sudo apt install -y groovy xsltproc xmlstarlet
```

Comment:
Is this more a temporary solution and that is why we aren't putting these into install scripts?

Reply:
No, we typically install dependencies in CI, other than the complicated ones below.